README.md

```diff
@@ -1,2 +1,36 @@
-# liquidator
+# umeeliqd
+
 A basic liquidator tool for Umee's native leverage protocol.
+
+```sh
+export UMEE_LIQUIDATOR_PASSWORD={keyring_password}
+$ umeeliqd --config {path/to/config.toml}
+```
+
+Uses a toml config file
+
+```toml
+[log]
+level="info"
+format="text"
```

Comment:
Is this all the configuration file contains? If so, we might just wanna stick with flags.

Reply:
For now, but we're going to need to put a lot of behavior-governing things (like desired liquidation rewards, transaction size minimums, etc) in there once the real code is in.

Reply:
Sounds good. And these can be provided via flags right (Just like the price-feeder)?

Reply:
Right now, config file is the only method since I'm working under the assumption that there will be no way to run the liquidator without one (given its future required fields).

Reply:
Yes, but we should be able to provide flags for some these such as log level and log format.

Reply:
I'll ask any remaining questions on our flags/env/file approach to the various configurations (including keyring passphrase) on the call today.